PWGCF/FemtoUniverse/TableProducer/femtoUniverseProducerTask.cxx

```diff
@@ -1069,6 +1069,23 @@ struct femtoUniverseProducerTask {
     }
   }
 
+  template <bool isMC, typename V0Type, typename TrackType,
+            typename CollisionType>
+  void fillCollisionsAndTracksAndV0AndPhiCentRun3(CollisionType const& col, TrackType const& tracks, V0Type const& fullV0s)
+  {
+    if (ConfIsRun3)
```

Comment:
I think that if you intend to use fillCollisionsAndTracksAndV0AndPhiCentRun3 you should not need ConfIsRun3, it could generate issues in the future.